ui/src/components/table/table-element/column-settings.tsx

```diff
@@ -0,0 +1,86 @@
+'use client';
```

Comment:
Could you put a newline under this, thanks.

Reply:
done